pkg/cloudprovider/fake/cloudprovider.go

```diff
@@ -161,6 +161,10 @@ func (c *CloudProvider) Delete(context.Context, *v1.Node) error {
 	return nil
 }
 
+func (c *CloudProvider) IsNodeDrifted(context.Context, *v1alpha5.Provisioner, *v1alpha1.Machine) (bool, error) {
```

Comment:
Why do we need the provisioner in this?
```suggestion
func (c *CloudProvider) IsMachineDrifted(context.Context, *v1alpha1.Machine) (bool, error) {
```

Reply:
You're right, we probably don't. Will remove.